src/components/shared/selector-modal/SelectorModal.test.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import SelectorModal, { ISelectorModalOption } from "./SelectorModal";
+import userEvent from "@testing-library/user-event";
+
+const onOptionSelectedMock = jest.fn();
```

Comment:
Is there not behavior that occurs after this function is called that can be asserted. Again any mocking and asserting of a function was called is code implementation detail testing and is something a User doesn't know or care about. 
@nobrayner @Aprillion

Reply:
If possible to test SelectorModal component by only testing the parent component, then we probably don't need a unit test of the component in isolation.

However, if we think of SelectorModal as a reusable component with "public" API that contains the `onOptionSelectedMock` prop, then it's appropriate to check that the public interface works as expected in the unit test.

So no strong opinion in either direction IMHO 🤷‍♂️